schema/crdb/README.adoc

```diff
@@ -53,11 +59,22 @@ Assumptions:
 
 Process:
 
-* Choose a `NEW_VERSION` number. This should almost certainly be a major version bump over `OLD_VERSION`.
-* Add a file to `schema/crdb/NEW_VERSION/up.sql` with your changes to the schema.
-** This file should validate the expected current version transactionally.
-** This file should only issue a single schema-modifying statement per transaction.
-** This file should not issue any data-modifying operations within the schema-modifying transactions.
+* Choose a `NEW_VERSION` number. This should almost certainly be a major
+  version bump over `OLD_VERSION`.
+* Create directory `schema/crdb/NEW_VERSION`.
+* If only one SQL statement is necessary to get from `OLD_VERSION` to
+  `NEW_VERSION`, put that statement into `schema/crdb/NEW_VERSION/up.sql`.  If
+  multiple statements are required, put each one into a separate file, naming
+  these `schema/crdb/NEW_VERSION/upN.sql` for as many `N` as you need.
+** Each file should contain _either_ one schema-modifying statement _or_ some
+   number of data-modifying statements.  You can combine multiple data-modifying
+   statements.  But you should not mix schema-modifying statements and
+   data-modifying statements in one file.  And you should not include multiple
+   schema-modifying statements in one file.
```

Comment:
Great summary, this is spot-on. 

Technically there are [exceptions](https://www.cockroachlabs.com/docs/stable/online-schema-changes#schema-changes-within-transactions):

> Some schema change operations can be run within explicit, multiple statement transactions. CREATE TABLE and CREATE INDEX statements can be run within the same transaction with the same atomicity guarantees as other SQL statements. There are no performance or rollback issues when using these statements within a multiple statement transaction.

But I think this is an apt summary, unless someone performing a schema change can cite more-specific docs from CRDB for a multi-statement transaction.